src/pages/home/components/Header/header.tsx

```diff
@@ -64,3 +64,4 @@ const RenderConnectBrightID = () => {
 };
 
 export default Header;
+export {Dabes}
```

Comment:
If you need to use one component in multiple places, please create a new file only for that component instead of exporting it like this.